ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -58,9 +58,10 @@ const AdvancedGasFeeGasLimit = () => {
   }
 
   return (
-    <Typography
-      tag={TypographyVariant.paragraph}
-      variant={TypographyVariant.H7}
+    <Text
+      tag={TextVariant.bodyMd}
+      variant={TextVariant.bodySm}
+      as="h6"
```

Comment:
This is the equivalent of `tag={TypographyVariant.paragraph}` for the Typography component
```suggestion
      as="p"
```